prow/plugins/plugins.go

```diff
@@ -509,6 +509,9 @@ type ConfigMapSpec struct {
 	// Namespace in which the configMap needs to be deployed. If no namespace is specified
 	// it will be deployed to the ProwJobNamespace.
 	Namespace string `json:"namespace,omitempty"`
+	// Namespaces in which the configMap needs to be deployed, in addition to the above
+	// namespace provided, or the default if it is not set.
+	AdditionalNamespaces []string `json:"additional_namespaces,omitempty"`
```

Comment:
why not just namespaces? Is there something special about the first one?

Reply:
Using `namespaces` and making it mutually exclusive with `namespace` seems like the best option to me. We could even load `namespace` into `namespaces` when parsing the config so that code doesn't have to consider both fields.
I'd like to avoid another `Refs` and `ExtraRefs` scenario (although that does have some reason to exist since the `Refs` is implicitly supplied by Prow while `ExtraRefs` is not).

Reply:
This is backwards compatible and requires no changes to any existing configurations or deployments. The first one _is_ special -- it is either empty (and defaulted) or explicit. If we had just a list of namespaces, we'd have to also have to have a flag on whether or not to add in the default.

Reply:
I guess I wanted to be able to specify _additional_ namespaces and not have to pass in the default explicitly, or risk a typo on the default, etc

Reply:
Backwards compat seems pretty easy regardless:

```
if namespace == "" { namespace = "default" } // technically shouldn't this be the same namespace as hoook?
if len(namespaces) == 0 { namespaces = []string{namespace} }
```

Reply:
Because if my intent as a user is to say "also deploy HERE" I should not be asked to also provide the previously implicit default

Reply:
Oh, you don't want to have to specify the default namespace. Interesting... Makes sense, especially naming it additional_namespaces